src/api/app/views/webui2/webui/groups/index.html.haml

```diff
@@ -4,3 +4,28 @@
 .card.mb-3
   = render partial: 'webui/configuration/tabs'
   .card-body
+    %h3 Manage Groups
+    %table.responsive.table.table-sm.table-striped.table-bordered.w-100#manage-groups-table
+      %thead
+        %tr
+          %th Group Name
+          %th Members
+          %th Action
+      %tbody
+        - @groups.each do |group|
+          %tr
+            %td
+              = link_to(group.title, group_show_path(title: group.title), id: group.title)
```

Comment:
[nitpick] `.title` is not needed (and in the rest of the places where it is converted to string or param neither